galata/test/documentation/debugger.test.ts

```diff
@@ -21,6 +21,16 @@ test.describe('Debugger', () => {
       [generateCaptureArea({ top: 62, left: 1050, width: 190, height: 28 })]
     );
 
+
+    let toResolve;
+    const awaitTimeout = new Promise(resolve => {
+      toResolve = resolve;
+    });
+
+    requestAnimationFrame(() => {
```

Comment:
Looks like this gives the following:

```
ReferenceError: requestAnimationFrame is not defined
```

Maybe need to be wrapped in a `page.evaluate()`?